### ISSUE TITLE
"Unnormalize", rescale Data

### ISSUE THREAD

**Issue by Steviey** [juanklopper/Deep-learning-using-R#1]
According to the code in "Implementing training improvements.R", ist there a way to "unormalize" rescale predictions made with normalized data, based on that implementation? 
I also tried this: https://keras.rstudio.com/reference/normalize.html#
But there seems to be no "retour" too. 
It seems there are cases where "unormalizing" is not necessary.
But in this case my predictions need it...
So far I've got this: https://github.com/rstudio/keras/issues/270 
Thank you in advanced, for this wonderful videos and documentations and greetings to Southafrica.

**Comment by Steviey** [comment]
No answer, too sad.
